client/snippets/snippets.json

```diff
@@ -364,5 +364,25 @@
         "prefix": "super",
         "body": "sendMessage(this, _CALL_ANCESTOR_OF, ${1:CURRENT_CLASS}, ${2:PARENT_CLASS}:::${3:PARENT_METHOD}, ${4:METHOD_ARGS});$0",
         "description": "Call a method of the parent class"
-    }
+    },
```

Comment:
Bad indentation settings. The file uses 4-space indentation and your edit uses hard tabs. It becomes quite apparent in GitHub's diff.